index.html

```diff
@@ -0,0 +1,227 @@
+
+<body>
+  <!--    bar menu and logo  -->
+  <header >
+    <section class="nav-bar-grey">
+      <a href="" class="nav-bar-grey-fabook"></a>
+      <a href="" class="nav-bar-grey-twitter"></a>
+      <a href="" class="nav-bar-grey-language">Language</a>
+      <a href="" class="nav-bar-grey-mypage">My Page</a>
+      <a href="" class="nav-bar-grey-logout">Logout</a>
+    </section>
+    <div class="nav-bar">
+      <section class="nav-menuContainer">
+        
+        <button class="bugerBtn">
+          <div class="burger-icon line1"></div>
+          <div class="burger-icon line2"></div>
+          <div class="burger-icon line3"></div>
+        </button>
+        <!--burger menu-->
+        <ul class="burger-menu">
+          <div class="burger-menu-style">
+              <li class="burger-menu-exit-button"><button class="exitButton"></button></li>
+              <li class="burger-menu-homePage-button"><a href="#home-page" class="burger-menu-home-page">Home Page</a></li>
+              <li class="burger-menu-aboutPage-button"><a href="#about-page" class="burger-menu-aboutPage-button">About</a></li>           
+              <li class="burger-menu-aboutPage-other"><a href="" class="burger-menu-aboutPage-other">Join</a></li>
+              <li class="burger-menu-aboutPage-other"><a href="" class="burger-menu-aboutPage-other">Sponsors</a></li>
+              <li class="burger-menu-aboutPage-other"><a href="" class="burger-menu-aboutPage-other">News</a></li> 
+          </div>
+        </ul>
+        <!---->
+        <ul class="desktop-menu">
+          <li><a href="#home-page" class="desktop-menu-home">Home</a></li>
+          <li><a href="#about-page" class="desktop-menu-about">About</a></li>
+          <li><a href="" class="desktop-menu-about">Join</a></li>
+          <li><a href="" class="desktop-menu-about">Sponsors</a></li>
+          <li><a href="" class="desktop-menu-about">News</a></li> 
+        </ul>        
+      </section>
+      <div class="nav-logo-container">
+        <a href="#home-page" class="nav-logo-link">
+          <div class="nav-logo-img"></div>
+            <h1 class="nav-logo">Libra</h1>
+        </a>
+      </div>
+    </div>
+  </header>
+
+  <!--    home page and about me page   -->
+
+  <main>
+
+    <!--    home page    -->
+    
+    <section class="home-page" id="home-page">
+
+      <!--    titles    -->
+    <section class="home-page-intro">
+        <div class="home-page-tittles">
+          <h2 class="home-page-h2">"Security is the future"</h2>
+          <h1 class="home-page-h1">Block-chain first TECH SUMMIT 2023</h1>
+        </div>
+
+        <!--    paragraph    -->
+
+        <div class="home-page-text">
+          <p class="home-page-text-p">Join us into the biggest congrest about block-chain technology
+            and know the new developments and improvements with companies
+            that are already developing applications that will renew the world
```

Comment:
- [ ] Good job here 👏 . But in comparison to the original design, kindly align the text to the left with lo as you can see on the design given. This makes its have a better visual appeal and a good user experience 👍 

## Your work:
![image](https://github.com/Zven94/conference-page/assets/50990925/0537c3e7-1a6c-4d18-b5b4-6e01c12d5092)

## Design given:
![image](https://github.com/Zven94/conference-page/assets/50990925/dc1d181c-3fa1-4278-bd8e-b785918f7316)